Store the Response from the API as a File / Added reload button

ronitjadhav opened the pull request:
#12 
- Store the response from the API as a file 
- Load from the file when opening the browser dialog
- Add a button to reload resources

ismailsunni commented:
I think you haven't added the part where the browser will load from the file. Or did I miss something?

Could you also put a screenshot of the new UI/dialog, when you made UI changes? I usually use Flameshot to take a screenshot (or any tool is fine)

ronitjadhav commented:
> I think you haven't added the part where the browser will load from the file. Or did I miss something?
> 
> Could you also put a screenshot of the new UI/dialog, when you made UI changes? I usually use Flameshot to take a screenshot (or any tool is fine)

1. The logic for the browser to load from the file is in the file api_client.py on line no: 17, 18, 19
2. Below is the screenshot of the new UI

![image](https://user-images.githubusercontent.com/9210179/236118403-27802002-9a93-477d-ae28-989891f774bb.png)

ronitjadhav commented:
@ismailsunni 
I have made changes mentioned by you:

1. Removed reload_resources() function, merged the function with populate_resources.
2. Updated the alignment of reload resources
3. Replaced push button with tool button and added an icon

![image](https://user-images.githubusercontent.com/9210179/236747469-9055c38f-585e-44cf-a694-1b13f196b75b.png)

ismailsunni reviewed (approved):
Thanks, Ronit, LGTM now.

There are some small tidy up that we can add later. I will create another ticket for it.
